Demo/MockVenmo/AppSwitcher.swift

```diff
@@ -5,63 +5,23 @@ enum AppSwitcher {
     static var openVenmoURL: URL?
 
     static var successURLWithPaymentContext: URL? {
-        var successComponents = openVenmoURL
-            .flatMap { URLComponents(url: $0, resolvingAgainstBaseURL: false) }?
-            .queryItems?
-            .first { $0.name == "x-success" }?
-            .value
-            .flatMap { URLComponents(string: $0) }
-
-        successComponents?.queryItems = [
-            URLQueryItem(name: "x-source", value: "Venmo"),
-            URLQueryItem(
-                name: "resource_id",
-                value: "cGF5bWVudGNvbnRleHRfZGNwc3B5MmJyd2RqcjNxbiM4NjE4ZThkYi0xZDJkLTQwYjktYWJjOC0zNTVlNTk5YzliNTg="
-            )
-        ]
-
-        return successComponents?.url
+        let resourceID = "cGF5bWVudGNvbnRleHRfZGNwc3B5MmJyd2RqcjNxbiM4NjE4ZThkYi0xZDJkLTQwYjktYWJjOC0zNTVlNTk5YzliNTg="
```

Comment:
Now that we only support universal links and these all used deeplinks they were failing. Updating these to the expected outputs with universal links allows these tests to pass again.